package.json

```diff
@@ -13,6 +13,7 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@gnosis.pm/safe-apps-provider": "^0.3.0",
```

Comment:
This new dependency isn't actually needed until the commented code is uncomented.